ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-service-insertion.yml

```diff
@@ -1,3 +1,12 @@
 ### Router service-insertion ###
 router_service_insertion:
   enabled: true
+  connections:
+    - connection_name: connection1
+      interface: ethernet2
+      next_hop_ip_address: '10.10.10.10'
```

Comment:
```suggestion
      next_hop: '10.10.10.10'
```

Reply:
only next hop will not give idea whether it is next-hop interface or ip.